### DISCUSSION TITLE
Using Groot2 in corporate setting / Adjusting the UI

### DISCUSSION THREAD

**Discussion by njoye** [BehaviorTree/BehaviorTree.CPP#780] [Q&A]
I would like to use Groot2 for certain tasks in our industrial/corporate setting. For that I would need to be able to adjust the UI though, so that we can have deeper integration with our ecosystem (think automatic deployment of trees, etc.). Is there a plan to enable end-users to add plugins or to modify the program behavior in the future? Or is something like this already possible ?

**Comment by facontidavide** [comment]
100% possible. Contact me on dfaconti@aurynrobotics.com
